snooty/rstspec.toml

```diff
@@ -8,6 +8,8 @@ guide_categories = ["Getting Started"]
 alignment = ["left", "center", "right"]
 card_type = ["small", "large"]
 output_format = ["html"]
+devhub_type = ["article", "quickstart", "how-to", "video", "live"]
+difficulty = ["beginner", "intermediate", "advanced"]
```

Comment:
If this is intended to be a distinct enum from `user_level`, could we prefix with `devhub_`?

Reply:
Oh duh. I'm happy to use the existing directive and delete `devhub:level`.